q2_feature_classifier/_cutter.py

```diff
@@ -6,11 +6,16 @@
 # The full license is in the file LICENSE, distributed with this software.
 # ----------------------------------------------------------------------------
 
-from itertools import chain
+import skbio
+import os
+from sklearn.externals.joblib import Parallel, delayed, effective_n_jobs
```

Comment:
see #134 . change to:
```from joblib import Parallel, delayed, effective_n_jobs```

Reply:
@nbokulich I made it `from sklearn.externals.joblib` not `from joblib` because `from joblib` wasn't working. After reading your comment, I tried changing it to `from joblib` again, and it did work. WHAT KIND OF BLACK MAGIC ARE YOU CONDUCTING